front50-core/src/main/java/com/netflix/spinnaker/front50/model/plugins/PluginInfoService.java

```diff
@@ -196,17 +205,34 @@ private void validate(PluginInfo pluginInfo) {
 
   private void cleanupPreferredReleases(PluginInfo pluginInfo, PluginInfo.Release release) {
     if (release.isPreferred()) {
+      String currentPreferredVersion =
+        pluginInfo.getReleases().stream().filter(PluginInfo.Release::isPreferred).findFirst().map(PluginInfo.Release::getVersion).orElse(null);
+
       Instant now = Instant.now();
       String user = AuthenticatedRequest.getSpinnakerUser().orElse("anonymous");
 
       pluginInfo.getReleases().stream()
-          .filter(it -> !it.getVersion().equals(release.getVersion()))
+          .filter(it -> !it.getVersion().equals(release.getVersion()) && it.isPreferred())
```

Comment:
Currently, all releases get "touched" when the preferred version is changed.  Now, only the new and old releases will get "touched".